vagrant/README.md

```diff
@@ -4,76 +4,43 @@ This is a fairly minimal Vagrantfile and associated bootstrap scripts for
 setting up a GCC-based build environment for
 [TrustedFirmware-M](https://www.trustedfirmware.org).
 
-Note that the vagrant virtual machine will contain an independent copy of TF-M
-and the mbed-os-tf-m-regression-tests.
-
-Note that by default vagrant shares `/vagrant` in the virtual machine with the
+Note that by default Vagrant shares `/vagrant` in the virtual machine with the
```

Comment:
I think this commit can be squashed with other commit changing the vagrant setup script.

Reply:
I've squashed it and updated the commit message